mpp/mpp.cpp

```diff
@@ -626,6 +638,9 @@ MPP_RET Mpp::control_mpp(MpiCmd cmd, MppParam param)
         MppPollType block = *((MppPollType *)param);
         mOutputBlock = block;
     } break;
+    case MPP_SET_OUTPUT_BLOCK_TIMEOUT: {
+        mOutputBlockTimeout = *((int *)param);
```

Comment:
If it is related to external standard usage or we can add a S32 to milisecond translation at the control?